packages/protocol/src/interfaces/IVault.sol

```diff
@@ -12,8 +12,14 @@ import {ILendingProvider} from "./ILendingProvider.sol";
 import {IFujiOracle} from "./IFujiOracle.sol";
 
 interface IVault is IERC4626 {
-  /// Events
+  struct RebalanceAction {
```

Comment:
maybe this can be handled in a dedicated PR

Reply:
I needed a place holder function to apply rebalancer role restriction. Do not intend to work on it in this PR. I added "TODO" statements.